test/models/piece_test.rb

```diff
@@ -1,7 +1,280 @@
 require 'test_helper'
 
-class PieceTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
-end
+class GameTest < ActiveSupport::TestCase
```

Comment:
Hey Stephen. I'm not sure why these weird code things are coming up. I
didn't see any of it in our code that I pushed. So by all means if u see
anything weird go ahead and change it.
On Aug 14, 2015 6:10 PM, "Stephen B. Thompson" <notifications@github.com>
wrote:

> In test/models/piece_test.rb
> <https://github.com/Rails-Riders/chess/pull/15#discussion_r37124271>:
>
> > @@ -1,7 +1,280 @@
> >  require 'test_helper'
> >
> > -class PieceTest < ActiveSupport::TestCase
> > -  # test "the truth" do
> > -  #   assert true
> > -  # end
> > -end
> > +class GameTest < ActiveSupport::TestCase
>
> On Line 3, can GameTest be changed back to the original PieceTest?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Rails-Riders/chess/pull/15/files#r37124271>.
>